.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+          go-version: '1.25'
+
+      - name: update-docs
+        run: |
+          ./generate_docs.sh
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add docs/commands/
+          git diff --staged --quiet || git commit -m 'chore: update CLI documentation'
+
+      - name: push-changes
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)